react_components/PaginationBoxView.js

```diff
@@ -61,6 +61,12 @@ export default class PaginationBoxView extends Component {
     }
   }
 
+  componentWillReceiveProps(nextProps) {
+    if (this.props.forceSelected !== nextProps.forceSelected) {
```

Comment:
shouldn't we also check if forceSelected is null/undefined or not, before invoking setState ?